README.md

```diff
@@ -37,12 +38,35 @@ regenerate Go code after you change the `sourcegraph.proto` file.
    git checkout proto3
    go install ./...
    ```
-1. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
+
+3. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
 
    ```
    go get -u sourcegraph.com/sourcegraph/gen-mocks
    ```
 
+4. **Install `gopathexec`**:
+
+   ```
+   go get -u github.com/shurcooL/gopathexec
+   ```
+
+#### OS X-specific
+
+```
+brew install gnu-sed --with-default-names
```

Comment:
That should work (assuming `sh` exists, should be safe on OS X and Linux, but not Windows), albeit being somewhat underhanded.

What's the advantage over doing `alias gsed='sed'` on Linux? Is it so that the Linux namespace doesn't need to be polluted with an extra command/alias?